MHArmory.Search/OpenCL/HostOptionsBuilder.cs

```diff
@@ -0,0 +1,29 @@
+using System.Text;
+
+namespace MHArmory.Search.OpenCL
+{
+    class HostOptionsBuilder
+    {
+        private StringBuilder InnerBuilder { get; }
+
+        public HostOptionsBuilder()
+        {
+            InnerBuilder = new StringBuilder();
+        }
+
+        public void AddDefine(string name)
+        {
+            InnerBuilder.Append($"-D {name} ");
+        }
+
+        public void AddDefine(string name, object value)
+        {
+            InnerBuilder.Append($"-D {name}={value} ");
+        }
+
+        public override string ToString()
+        {
+            return InnerBuilder.ToString(0, InnerBuilder.Length - 1);
+        }
+    }
+}
```

Comment:
Please make sure all files have line endings.

Reply:
I didn't quite understand, do you mean `\r\n` vs `\n`? If so, which one should I prefer?

Reply:
I always use `\n` (unless mistake), but what I mean is I want all files to end with a carriage return (`\n`).